[2024-05-05 09:50:39,247] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-05-04T00:00:00+00:00 [queued]>
[2024-05-05 09:50:39,252] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-05-04T00:00:00+00:00 [queued]>
[2024-05-05 09:50:39,252] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 09:50:39,252] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-05-05 09:50:39,252] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 09:50:39,274] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): checking_raw_data_schema> on 2024-05-04 00:00:00+00:00
[2024-05-05 09:50:39,278] {standard_task_runner.py:52} INFO - Started process 1155 to run task
[2024-05-05 09:50:39,283] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'checking_raw_data_schema', 'scheduled__2024-05-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/.ipynb_checkpoints/lead_scoring_data_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmpunwm9xwh', '--error-file', '/tmp/tmpmh7px1ue']
[2024-05-05 09:50:39,284] {standard_task_runner.py:80} INFO - Job 6: Subtask checking_raw_data_schema
[2024-05-05 09:50:39,338] {task_command.py:371} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-05-04T00:00:00+00:00 [running]> on host 95e735c85bd1
[2024-05-05 09:50:39,405] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_Scoring_Data_Engineering_Pipeline
AIRFLOW_CTX_TASK_ID=checking_raw_data_schema
AIRFLOW_CTX_EXECUTION_DATE=2024-05-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-04T00:00:00+00:00
[2024-05-05 09:50:43,179] {logging_mixin.py:115} INFO - Raw datas schema is NOT in line with the schema present in schema.py
[2024-05-05 09:50:43,234] {python.py:173} INFO - Done. Returned value was: None
[2024-05-05 09:50:43,241] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=checking_raw_data_schema, execution_date=20240504T000000, start_date=20240505T095039, end_date=20240505T095043
[2024-05-05 09:50:43,308] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-05-05 09:50:43,421] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
